apps/test-suite/screens/SelectScreen.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import {
+  TouchableHighlight,
+  TouchableNativeFeedback,
+  StyleSheet,
+  FlatList,
+  View,
+  Text,
+  Platform,
+  Linking,
+  Alert,
+  Button,
+  PixelRatio,
+} from 'react-native';
+import { SafeAreaView } from 'react-navigation';
+import { MaterialIcons } from '@expo/vector-icons';
+import { getTestModules } from '../TestUtils';
+
+class ListItem extends React.Component {
+  onPress = () => {
+    this.props.onPressItem(this.props.id);
+  };
+
+  renderView = () => {
+    const checkBox = this.props.selected ? 'check-box' : 'check-box-outline-blank';
+    return (
+      <View style={styles.listItem}>
+        <MaterialIcons name={checkBox} size={26} />
+        <Text style={styles.label}>{this.props.title}</Text>
+      </View>
+    );
+  };
+
+  render() {
+    return Platform.select({
+      android: (
+        <TouchableNativeFeedback onPress={this.onPress}>
+          {this.renderView()}
+        </TouchableNativeFeedback>
+      ),
+      default: (
+        <TouchableHighlight onPress={this.onPress} underlayColor="lightgray">
+          {this.renderView()}
+        </TouchableHighlight>
+      ),
+    });
+  }
+}
+
+export default class SelectScreen extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.modules = getTestModules();
+    this.state = {
+      selected: new Set(),
+    };
+  }
+
+  componentDidMount() {
+    Linking.getInitialURL()
+      .then(url => {
+        if (url && url.indexOf('/all') > -1) {
```

Comment:
let's add a `TODO` comment here about replacing this logic once we have `Linking.parse` on web